rdf/rdf12/rdf-n-quads/syntax/nquads-star-syntax-4.nq

```diff
@@ -1 +1 @@
-<<<http://example/s1><http://example/p1><http://example/o1>>><http://example/q><<<http://example/s2><http://example/p2><http://example/o2>>><http://example/g>.
+<<(<http://example/s1><http://example/p1><http://example/o1>)>><http://example/q><<(<http://example/s2><http://example/p2><http://example/o2>)>><http://example/g>.
```

Comment:
The Manifest needs to be updated for this to be a Negative Syntax Test.

Reply:
Yeah, good catch. I just did a search and replace, and hadn't pulled back far enough to recognize the issue that introduced in these tests.